internalshared/configutil/telemetry.go

```diff
@@ -35,6 +37,11 @@ type Telemetry struct {
 	DisableHostname     bool   `hcl:"disable_hostname"`
 	EnableHostnameLabel bool   `hcl:"enable_hostname_label"`
 	MetricsPrefix       string `hcl:"metrics_prefix"`
+	ClusterName         string `hcl:"cluster_name"`
```

Comment:
Scott,

I am really sorry to have not made this clearer (I only left a comment in VLT-099 rather than editing it)  I think this attribute is unnecessary since the config already includes a cluster name elsewhere; it doesn't seem necessary to allow  telemetry to have a different user-provided name?  I just wanted to avoid the machine-generated one.

What do you think?